test/e2e/openLibraryLogo.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { test, expect, Page } from "@playwright/test";
+import { OpenLibraryLogo } from "../pages/OpenLibraryLogo";
+import { BASE_URL } from "../helper-config";
+
+let openLibraryLogoPage: OpenLibraryLogo;
+
+test.beforeEach(async({ page }: {page: Page}) => {
+    openLibraryLogoPage = new OpenLibraryLogo({page});
+    await openLibraryLogoPage.navigateBasePage();
+})
+
+test("check open-library logo is visible", async({page}: {page: Page}) => {
```

Comment:
It is already mentioned as `test` so we don't need to add `check` in the test description. Please follow same for other test cases too.
```suggestion
test("open-library logo", async({page}: {page: Page}) => {
```